### PR TITLE
build: :white_check_mark: load all fixtures, before running tests

### PR DESCRIPTION
**Pull request by lwjohnst86**
## Description

- This PR updates the justfile to load all JSON files found in fixture folders, rather than one by one (if there are more). This also makes the justfile more generic for other projects too.

## Testing

- [x] Yes
- [ ] No, not needed (give a reason below)
- [ ] No, I need help writing them

## Reviewer Focus

This PR only needs a quick review.

### CODE REVIEW

File: common/justfile/django [hunk 1]
```diff
@@ -44,6 +44,6 @@ install-deps:
   # no-root to not install the parent folder as a package
   poetry install --no-root
 
-# Add test data when running locally based on app/fixtures/sample.json
+# Add test data when running locally based on json files found in `fixtures/`
 add-test-data: install-deps update-migrations
-  poetry run python manage.py loaddata sample
+  poetry run python manage.py loaddata */fixtures/*.json
```
Comment (philter87) [inline common/justfile/django:49]: Nice!

File: common/justfile/django [hunk 2]
```diff
@@ -23,7 +23,7 @@ update-migrations: install-deps
   poetry run python manage.py migrate
 
 # Run Django tests
-run-tests: install-deps update-migrations
+run-tests: install-deps update-migrations add-test-data
```
Comment (philter87) [inline common/justfile/django:26]: But I don't think this is necessary. The tests are running against another temporary database, so sample data will not be visible in the tests. 

We should use fixtures more in tests. Fixtures can easily be applied in unit tests, see https://django-testing-docs.readthedocs.io/en/latest/fixtures.html#fixtures-in-unit-tests

**Review by philter87 — COMMENTED** [review]
Nice with a pattern :)

File: common/justfile/django (hunk 2 above)
Comment (lwjohnst86) [reply common/justfile/django:26]: Ok! I will remove that commit than ☺️

**Pull request merged**